### PR TITLE
[14.0][IMP] project_milestone: enable chatter

### PR DESCRIPTION
**Pull request by lanto-razafindrabe**

### CODE REVIEW

**Comment by rvalyi** [comment]
could you squash the 3 last commits together?

**Comment by lanto-razafindrabe** [comment]
> could you squash the 3 last commits together?

Alright. Thank you.

**Comment by lanto-razafindrabe** [comment]
@rvalyi Have you an idea why the codecov/project is failed on checks ?

Thank you.

**Comment by TumbaoJu** [comment]
@OCA/project-service-maintainers Hello! Could you please take a look at this PR? It would be much appreciated.
Thank you!

**Review by dreispt — APPROVED** [review]

**Comment by TumbaoJu** [comment]
Thank you @dreispt !

**Review by leemannd — APPROVED** [review]

File: project_milestone/__manifest__.py [hunk 1]
```diff
@@ -6,9 +6,9 @@
     "author": "Patrick Wilson, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/project",
     "category": "Project Management",
-    "version": "14.0.1.3.0",
+    "version": "14.0.1.3.1",
```
Comment (grindtildeath) [inline project_milestone/__manifest__.py]: IMO This is more than just a patch, must at lease increase the minor version.

Comment (lanto-razafindrabe) [reply project_milestone/__manifest__.py]: Hi @grindtildeath 

I have changed it to "14.0.1.4.0".

Thank you.

**Comment by TumbaoJu** [comment]
@OCA/project-service-maintainers : Can someone help us with this PR.  We don't know why the `codecov/project` has failed.

Can this PR be merge anyway or if not, what do we have to change?

Thank you in advance.
@lanto-razafindrabe : FYI

**Comment by TumbaoJu** [comment]
@dreispt  Thank you!
@lanto-razafindrabe @majouda  PR merged!

**Pull request merged**